ui/components/app/collectibles-detection-notice/collectibles-detection-notice.js

```diff
@@ -9,54 +9,104 @@ import {
   TEXT_ALIGN,
   FONT_WEIGHT,
   DISPLAY,
+  FLEX_DIRECTION,
+  BLOCK_SIZES,
 } from '../../../helpers/constants/design-system';
 import { useI18nContext } from '../../../hooks/useI18nContext';
 import Button from '../../ui/button';
 import { EXPERIMENTAL_ROUTE } from '../../../helpers/constants/routes';
 import { setCollectiblesDetectionNoticeDismissed } from '../../../store/actions';
+import { ENVIRONMENT_TYPE_FULLSCREEN } from '../../../../shared/constants/app';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
 
 export default function CollectiblesDetectionNotice() {
   const t = useI18nContext();
   const history = useHistory();
+  const environmentType = getEnvironmentType();
 
   return (
-    <Box marginBottom={4} className="collectibles-detection-notice">
+    <Box className="collectibles-detection-notice">
       <Dialog type="message" className="collectibles-detection-notice__message">
-        <button
-          onClick={() => setCollectiblesDetectionNoticeDismissed()}
-          className="collectibles-detection-notice__message__close-button"
-          data-testid="collectibles-detection-notice-close"
-        />
-        <Box display={DISPLAY.FLEX}>
-          <Box paddingTop={2}>
-            <i style={{ fontSize: '1rem' }} className="fa fa-info-circle" />
+        {environmentType !== ENVIRONMENT_TYPE_FULLSCREEN && (
+          <button
+            onClick={() => setCollectiblesDetectionNoticeDismissed()}
+            className="fas fa-times collectibles-detection-notice__message__close-button"
+            data-testid="collectibles-detection-notice-close"
+          />
+        )}
+        <Box display={DISPLAY.FLEX} width={BLOCK_SIZES.MAX}>
+          <Box paddingTop={1}>
+            <i
+              style={{ fontSize: '1rem', color: '#037DD6' }}
```

Comment:
Use the dedicated css vars outlined in the [color doc](https://metamask.github.io/metamask-storybook/index.html?path=/story/design-tokens-color--page#theme-colors-tier-2)

```suggestion
              style={{ fontSize: '1rem', color: 'var(--color-primary-default)' }}
```